test/library/packages/UnitTest/test_locales.chpl

```diff
@@ -22,4 +22,26 @@ proc s4(test: Test) throws {
   test.minLocales(10);
 }
 
-UnitTest.runTest(s1,s2,s3,s4);
+// This test has only max locales
+proc s5(test: Test) throws {
+  test.maxLocales(5);
+}
+
+//This test has only min locales
+proc s6(test: Test) throws {
+  test.minLocales(6);
+}
+
+
+//This test has addNumLocales Multiple Times
+proc s7(test: Test) throws {
+  test.addNumLocales(8,16);
+  test.addNumLocales(16,7);
```

Comment:
It would be good to have a separate test for the "out-of-order" case in addition to this test.

Reply:
Ma'am, I made s3 to be out of order.

Reply:
Oh, dur.  But then you don't have a check of in order . . .

Reply:
It's probably fine

Reply:
I will create an in-order test as well.